versioning-strategies/accept-version.js

```diff
@@ -0,0 +1,9 @@
+const SemVerStore = require('semver-store')
```

Comment:
Can you add an `'use strict'` at the top?

Reply:
done, thanks